ODIN_II/SRC/netlist_statistic.cpp

```diff
@@ -186,6 +205,9 @@ static void count_node_type(operation_list op, nnode_t* node, netlist_t* netlist
         case CLOCK_NODE:
         case FF_NODE:
         case MULTIPLY:
+            increment_type_count(op, netlist);
```

Comment:
isnt the fallthrough doing this already?

Reply:
Fixed, thank you